apps/backend/src/libs/modules/encryption/encryption.ts

```diff
@@ -0,0 +1,8 @@
+import { config } from "~/libs/modules/config/config.js";
+
+import { BaseEncryption } from "./base-encryption.module.js";
+
+const encrypt = new BaseEncryption(config);
```

Comment:
Let's not load the whole config but pass only necessary arguments to the constructor in a manner like `{salt: SALT_FROM_ENV, etc.}`